### DISCUSSION TITLE
CAVE

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1479] [Icon request] [answered]
![Screenshot 2024-01-09 at 8 10 24 PM](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/96893300/ca93249c-b70a-46c1-a718-e529e1465085)

Can anyone make an icon for AWIPS Cave?

**Comment by Karukhai** [comment, marked as answer]
Hi, there. It's currently waiting for approval, but here's the PNG:

![CAVE](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/42078366/0f699c76-75c6-4359-840b-149e3d5c6fe9)

Here's my profile if you want to download the .icns file once it is approved: [https://macosicons.com/#/u/kenji](https://macosicons.com/#/u/kenji)
